_pandoc_filter/add_title.lua

```diff
@@ -0,0 +1,317 @@
+function fix_path (path)
+  return '.' .. path
+end
+
+
+function skip_include(el)
+
+	for i, item in ipairs(el) do
+				el[i] = pandoc.walk_block(item, {
+      Image = function (element)
+	        element.src = fix_path(element.src)
+            return element
+		   end
+		   })
```

Comment:
IDK lua, but are there usually standards for indenting? This is all over the place.